Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -1,18 +1,23 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
 using Promact.Trappist.DomainModel.ApplicationClasses.Question;
 using Promact.Trappist.DomainModel.Enum;
 using Promact.Trappist.Repository.Questions;
+using Promact.Trappist.Web.Models;
 using System.Threading.Tasks;
 namespace Promact.Trappist.Core.Controllers
 {
     [Route("api/question")]
     public class QuestionController : Controller
     {
         private readonly IQuestionRespository _questionsRepository;
+        private readonly UserManager<ApplicationUser> _userManager;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_userManager" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)